src/Shared/EmptyState.js

```diff
@@ -72,47 +86,70 @@ export default class EmptyListWrapper extends Component {
             </View>
           </View>
         )}
-      </>
+      </View>
     )
   }
 }
 
 function ImageHolder(props) {
-  let { imageSource, emptyStateImageStatus } = props
+  let {
+    imageSource,
+    emptyStateImageStatus,
+    textTitleDisplay,
+    imageWidth,
+    imageHeight,
+    imageRounding,
+  } = props
   let realImageSource = !imageSource
     ? require('./sqr-empty-state.png')
     : imageSource
+
+  const imageWrapperSize = { width: imageWidth, height: imageHeight }
+  const imageBorderRadius = {
+    borderRadius:
+      imageHeight < imageWidth
+        ? (imageHeight / 2) * (imageRounding / 100)
+        : (imageWidth / 2) * (imageRounding / 100),
+  }
+
   if (!emptyStateImageStatus || emptyStateImageStatus === 'noImage') {
     return (
       <>
         <TitleHolder {...props}></TitleHolder>
       </>
     )
   } else if (emptyStateImageStatus === 'above') {
+    const imageMargin = { marginBottom: textTitleDisplay !== 'noText' ? 32 : 0 }
     return (
       <>
         <View style={styles.emptyList}>
-          <Image
-            resizeMode="cover"
-            style={styles.image}
-            source={realImageSource}
-            pointerEvents="none"
-          />
+          <View style={[imageWrapperSize, imageMargin]}>
+            <Image
+              resizeMode="cover"
+              style={[styles.image, imageBorderRadius]}
+              source={realImageSource}
+              pointerEvents="none"
+            />
+          </View>
         </View>
         <TitleHolder {...props}></TitleHolder>
       </>
     )
   } else {
+    const imageMargin = { marginTop: textTitleDisplay !== 'noText' ? 32 : 0 }
```

Comment:
You could probably refactor the `textTitleDisplay` part of that code further up instead of repeating it.

Reply:
I'm not sure I follow here, like create a `shouldHaveMargin` variable higher up and use that when calculating the image margin?

Reply:
Yeah, never mind on this one. I got a little too nitpicky